cns/restserver/util.go

```diff
@@ -606,7 +608,9 @@ func (service *HTTPRestService) attachOrDetachHelper(req cns.ConfigureContainerN
 				}
 				nmaNCs := map[string]string{}
 				for _, nc := range ncVersionListResp.Containers {
-					nmaNCs[nc.NetworkContainerID] = nc.Version
+					// store nmaNCID as lower case to allow case insensitive comparison with nc stored in CNS
+					nmaNCID := cns.SwiftPrefix + strings.ToLower(nc.NetworkContainerID)
```

Comment:
this is introducing a bug,
In the original code, the swift prfix wasn't there. Why are you adding it?